setup.py

```diff
@@ -65,8 +65,8 @@ def get_dist(pkgname):
     packages=['visdom'],
     package_dir={'': 'py'},
     package_data={'visdom': ['static/*.*', 'static/**/*', 'py.typed', '*.pyi']},
-    include_package_data=True,
-    zip_safe=False,
+    include_package_data=bool(1),
+    zip_safe=bool(0),
```

Comment:
How is this an improvement?

Reply:
@JackUrb In this instead of True or False I used bool(1) or bool(0) respectively. For better clarity and faster processing of program used this.

Reply:
I'm entirely certain that `True` and `False` are clearer and more readable than `bool(1)` and `bool(0)`, and I'd wager making this call is less efficient. Can you point to any resource that suggests that replacing the python bool constant with _a bool cast on an int_ would be faster?

Reply:
No I don't have any resource that points out this . I learnt in my school days with my teacher that it's good to use bool(1) or bool(0) over True or False that sounds good in terms of technical aspects.

Reply:
I'd refrain from applying that idea to Python in the future - you can always test this out yourself to see it's not true:
```
>>> import timeit
>>> from numpy import mean
>>> def check_false_speed():
...   if False:
...     return "hi"
...   else:
...     return "bye"
... 
>>> def check_bool_speed():
...   if bool(0):
...     return "hi"
...   else:
...     return "bye"
... 
>>>  mean(timeit.repeat(check_false_speed, repeat=10))
0.05913775699999633
>>> mean(timeit.repeat(check_bool_speed, repeat=10))
0.09170382250000557
```

Reply:
Okk. I got it . Thanks @JackUrb